test/features/resource_availability.feature

```diff
@@ -16,184 +16,166 @@ Feature: Re-identification risk governance acknowledgement or Resource visibilit
+        Then I should not see "Visibility/Governance Acknowledgment"
+        Then I should see "Resource visible"
+        Then I should see "Re-identification risk governance completed?" 
 
         Examples: Users
             | User          |
             | SysAdmin      |
             | TestOrgAdmin  |
             | TestOrgEditor |
 
-
-    Scenario Outline: User creates resource for a dataset with Contains de-identified data is NO
-        Given "<User>" as the persona
-        When I log in
-
         ###
-        # Create resource that available for non-logged in user.
+        # Resource not available for non-logged users
         ###
-        And I go to "/dataset/new_resource/contains-de-identified-data-no"
-
-        # Check field visibility, xpath element start with 1.
-        Then I should see an element with id "field-resource_visibility"
-        Then I should see an element with xpath "//select[@id='field-resource_visibility']/option[3][@disabled]"
-
-        # Create the resource, with success.
-        Then I execute the script "document.getElementById('field-image-url').value='http://ckanext-data-qld.docker.amazee.io/'"
-        Then I fill in "name" with "resource created by <User> and is available"
-        Then I fill in "description" with "description"
-        Then I fill in "size" with "1024"
-        Then I select "Resource visible and re-identification risk governance acknowledgement not required" from "resource_visibility"
-        Then I press "save"
-        And I wait for 10 seconds
-        Then I should see "resource created by <User> and is available"
-
+        Given "Unauthenticated" as the persona
+        And I go to "/contains-de-identified-data-yes-visibility-true-acknowledged-no"
+        Then I should not see "Resource hidden"
+        
         ###
-        # Create resource that available for non-logged in user with resource_visibility blank.
+        # Resource not available for non-logged users
         ###
-        And I go to "/dataset/new_resource/contains-de-identified-data-no"
+        Given "Unauthenticated" as the persona
+        And I go to "/contains-de-identified-data-no-visibility-false-acknowledged-no"
+        Then I should not see "Resource hidden"
 
-        # Create the resource, with success.
-        Then I execute the script "document.getElementById('field-image-url').value='http://ckanext-data-qld.docker.amazee.io/'"
-        Then I fill in "name" with "resource created by <User> and is available with blank resource_visibility"
-        Then I fill in "description" with "description"
-        Then I fill in "size" with "1024"
-        Then I press "save"
-        And I wait for 10 seconds
-        Then I should see "resource created by <User> and is available with blank resource_visibility"
+        ###
+        # Resource available for non-logged users
+        ###
+        Given "Unauthenticated" as the persona
```

Comment:
Not sure what the point of the "Unauthenticated" persona is, since we'll get an error anyway if we try to use it to log in.

Reply:
@ThrawnCA  on https://github.com/salsadigitalauorg/ckanext-data-qld/pull/65#discussion_r840253410 if resource_visible is false it is hidden.
the other state is if governance acknowledgement is not in correct state and dataset has flag set.